File: HW01/CPP/src/CampingStove.cpp
#include "../include/CampingStove.h"
#include <iostream>
#include <string>
using namespace std;

CampingStove::CampingStove(string size, int wattage, double price) : OutdoorItem(price){
    this->size = size;
    this->wattage = wattage;
    this->price = price;
}

void CampingStove::display() {
    cout << "$" << this->price << " - This camping stove is size " << this->size << " and has a wattage of " << this->wattage << endl;
}

File: HW01/PYTHON/project/CampingStove.py
from .OutdoorItem import OutdoorItem

class CampingStove(OutdoorItem):
    def __init__(self, price, wattage, size):
        super().__init__(price)
        self.wattage = wattage
        self.size = size
        
    def display(self):
        print(f"${self.price} - This camping stove is size {self.size} and has a wattage of {self.wattage}", end="")